src/ui/views/EditAwards.tsx

```diff
@@ -0,0 +1,428 @@
+import React, { FormEvent, useState, useEffect } from "react";
+
+import useTitleBar from "../hooks/useTitleBar";
+import type { Player, View } from "../../common/types";
+import { logEvent, toWorker, helpers, realtimeUpdate } from "../util";
+import SelectReact from "../components/SelectMultiple";
+const EditAwards = ({
+	godMode,
+	players,
+	awards,
+	season,
+}: View<"editAwards">) => {
+	useTitleBar({
+		title: "Edit awards",
+		dropdownView: "edit_awards",
+		dropdownFields: { seasons: season },
+	});
+	const awardsInitial = awards;
+	const football = process.env.SPORT === "football";
+	const basketball = process.env.SPORT === "basketball";
+	const [aws, setAws] = useState(() => helpers.deepCopy(awards));
+	useEffect(() => {
+		setAws(() => helpers.deepCopy(awards));
+	}, [awards, season]);
+	const handleChange = (obj: any, pl: any) => {
+		let error: boolean = false;
+		const p: any = pl;
+		const type = obj.props.award;
+		const numberTeam = obj.props.teamNumber;
+		const numberPlayer = obj.props.playerNumber;
+
+		setAws((prevState: any) => {
+			const aws: any = prevState;
+
+			if (
+				type == "finalsMvp" ||
+				type == "mvp" ||
+				type == "smoy" ||
+				type == "roy" ||
+				type == "mip" ||
+				type == "oroy" ||
+				type == "droy"
+			) {
+				if (p.pid == undefined) {
+					aws[type] = undefined;
+				} else {
+					if (basketball) {
+						aws[type] = {
+							pid: p.pid,
+							name: p.name,
+							tid: p.tid,
+							abbrev: p.abbrev,
+							pts: p.currentStats == undefined ? 0.0 : p.currentStats.pts,
```

Comment:
(don't fix this, I'm editing stuff now)

0.0 vs 0 in Python is meaningful, but JS has only floats, so 0 and 0.0 are exactly the same thing.